sentences/fr/vacuum_HassVacuumReturnToBase.yaml

```diff
@@ -0,0 +1,8 @@
+language: fr
+intents:
+  HassVacuumReturnToBase:
+    data:
+      - sentences:
+          - "renvoie [<le>]{name} [(à|sur) (sa|la) base]"
```

Comment:
Same comment as for HassSetPosition : Verbs should be expansion rules and support infinitive forms